src/app_background.rs

```diff
@@ -38,10 +42,18 @@ impl AppBackground {
     }
 }
 
-fn load_image_from_path(path: &std::path::Path) -> Result<egui::ColorImage, image::ImageError> {
+fn load_image_from_path(
+    path: &std::path::Path,
+    alpha: u8,
+) -> Result<egui::ColorImage, image::ImageError> {
     let image = image::io::Reader::open(path)?.decode()?;
     let size = [image.width() as _, image.height() as _];
-    let image_buffer = image.to_rgba8();
+    let mut image_buffer = image.to_rgba8();
+
+    for pixel in image_buffer.pixels_mut() {
+        pixel.0[3] = alpha;
```

Comment:
Our image could have alpha already, couldn't it? Shouldn't we multiply by the alpha and then divide by 255/256 (don't know exactly which of the two) instead?